test/k8s-integration/driver-config.go

```diff
@@ -127,13 +127,9 @@ func generateDriverConfigFile(testParams *testParameters) (string, error) {
 		snapshotClassName = "no-volumesnapshotclass"
 	}
 
-	caps = append(caps, "pvcDataSource")
+	//caps = append(caps, "pvcDataSource")
 	minimumVolumeSize := "5Gi"
 	numAllowedTopologies := 1
-	if testParams.storageClassFile == regionalPDStorageClass {
```

Comment:
I don't think we should tear this out?

Reply:
pd-extreme does not support regional pd, I just remove this for testing, will revert this at submit.